### ISSUE TITLE
[Bug]: Crashes on World generation

### ISSUE THREAD

**Issue by rainysux** [Alex-the-666/Ice_and_Fire#4982]
### What happened?

When creating or joining premade world

### Minecraft Version

1.20.1

### Ice and Fire version

iceandfire-2.1.13-1.20.1+build.2.1.13-alpha-3

### Citadel version

 citadel-2.4.5-1.20.1.jar

### Relevant log output

```shell
Crash Report: https://pastebin.com/SFBmncTr
Latest Log: https://pastebin.com/jtrrLfU5
```

### Other Information

- [X] I am using other mods besides Ice and Fire
- [X] I have attached a log file from the session that I have experienced this issue

**Comment by TheBv** [comment]
Which version of JEI are you using? (related #4969)

**Comment by rainysux** [comment]
I am using REI actually

**Comment by rainysux** [comment]
looks like it works when using jei

**Comment by TheBv** [comment]
Alright I see what the problem is now.
I fixed it in 97d6f5bd922721d5d0e2038a4a34d8bb8cdb8dd1
